ui/apps/platform/src/Containers/AccessControl/AuthProviders/RuleGroups.tsx

```diff
@@ -155,6 +172,18 @@ function RuleGroups({
                                             <TrashIcon />
                                         </Button>
                                     </FlexItem>
+                                ) : (
+                                    <FlexItem>
+                                        <Tooltip content="This rule is managed declaratively and cannot be edited.">
```

Comment:
nit: should we specify that this can _only_ be modified within the "declarative" workflow, i.e something like:
`This rule is managed declaratively and can only be edited declaratively`?